modules/ossm-security-mtls.adoc

```diff
@@ -61,3 +61,38 @@ spec:
     tls:
       mode: ISTIO_MUTUAL
 ----
+
+== Setting the minimum and maximum protocol versions
+
+If your environment has specific requirements for encrypted traffic in your service mesh, such as for GDPR, you can control the cryptographic functions that are allowed by setting the `spec.istio.global.tls.minProtocolversion` or `spec.istio.global.tls.maxProtocolversion` in your ServiceMeshControlPlane resource. Those values define the minimum and maximum TLS version in your control plane to use minimum or maximum level of TLS across all encrypted traffic in your mesh.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    global:
+      tls:
+        minProtocolversion: TLSv1_0
+----
+
+The valid values correspond to TLS versions. The default is `auto` and does not specify a version of TLS.
+
+.Valid values
+|===
+|Value|Description|
+| `auto`
+| default
+
+`TLSv1_0`
```

Comment:
This format appears to be incorrect

Reply:
What is the correct format. I may have copied and pasted the wrong item from bluejeans.

Reply:
Sorry, I meant the markdown for the table is incorrect and it is not rendering properly.  If you check the netlify output you should be able to see how these cells are being merged incorrectly.

Reply:
|`TLSv1_0`